compiler/modules/bir/verify.bal

```diff
@@ -67,18 +67,59 @@ class VerifyContext {
 
 public function verifyFunctionCode(Module mod, FunctionDefn defn, FunctionCode code) returns Error? {
     VerifyContext cx = new(mod, defn);
+    boolean[] tmpRegisterInit = [];
```

Comment:
On a second though, let's call this `tmpRegisterInitialized` to match with the func name.